wgpu-hal/src/vulkan/instance.rs

```diff
@@ -16,13 +16,30 @@ unsafe extern "system" fn debug_utils_messenger_callback(
     callback_data_ptr: *const vk::DebugUtilsMessengerCallbackDataEXT,
     _user_data: *mut c_void,
 ) -> vk::Bool32 {
-    const VUID_VKSWAPCHAINCREATEINFOKHR_IMAGEEXTENT_01274: i32 = 0x7cd0911d;
     use std::borrow::Cow;
 
     if thread::panicking() {
         return vk::FALSE;
     }
 
+    let cd = unsafe { &*callback_data_ptr };
+
+    const VUID_VKSWAPCHAINCREATEINFOKHR_IMAGEEXTENT_01274: i32 = 0x7cd0911d;
+    const VUID_VKCMDENDDEBUGUTILSLABELEXT_COMMANDBUFFER_01912: i32 = 0x56146426;
+
+    // Silence Vulkan Validation error " VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912"
+    // This is a bug in the validation layer:
+    // https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5671
+    if cd.message_id_number == VUID_VKCMDENDDEBUGUTILSLABELEXT_COMMANDBUFFER_01912 {
+        return vk::FALSE;
+    }
```

Comment:
If we can get the version of the validation layers, it would be better to conditionally ignore it.

Reply:
To narrow down the affected Mesa versions: it seems like the validation was introduced in KhronosGroup/Vulkan-ValidationLayers#5277, merged 2023-2-16.